default.nix

```diff
@@ -78,12 +78,12 @@ in rustPlatform.buildRustPackage {
   # cp: to copy str.zig,list.zig...
   # wrapProgram pkgs.stdenv.cc: to make ld available for compiler/build/src/link.rs
   postInstall = if pkgs.stdenv.isLinux then ''
-    cp -r target/x86_64-unknown-linux-gnu/release/lib/. $out/lib
+    cp -r target/x86_64-unknown-linux-gnu/release/lib* $out/lib
     wrapProgram $out/bin/roc --set NIX_GLIBC_PATH ${nixGlibcPath} --prefix PATH : ${
       pkgs.lib.makeBinPath [ pkgs.stdenv.cc ]
     }
   '' else ''
-    cp -r target/aarch64-apple-darwin/release/lib/. $out/lib
+    cp -r target/aarch64-apple-darwin/release/lib* $out/lib
```

Comment:
not 100% sure about these, it's just something that kinda works.

The `$out/lib` directory would contain files like these:

```
libarena_pool.d
libarena_pool.rlib
libmorphic_lib.d
libmorphic_lib.rlib
librepl_test.d
librepl_test.rlib
libroc_alias_analysis.d
libroc_alias_analysis.rlib
libroc_ast.d
libroc_ast.rlib
libroc_build.d
libroc_build.rlib
libroc_builtins.d
libroc_builtins.rlib
libroc_can.d
libroc_can.rlib
libroc_cli.d
libroc_cli.rlib
libroc_code_markup.d
libroc_code_markup.rlib
libroc_collections.d
libroc_collections.rlib
libroc_constrain.d
libroc_constrain.rlib
libroc_debug_flags.d
libroc_debug_flags.rlib
libroc_derive.d
libroc_derive.rlib
libroc_derive_key.d
libroc_derive_key.rlib
libroc_docs.d
libroc_docs.rlib
libroc_editor.d
libroc_editor.rlib
libroc_error_macros.d
libroc_error_macros.rlib
libroc_exhaustive.d
libroc_exhaustive.rlib
libroc_fmt.d
libroc_fmt.rlib
libroc_gen_dev.d
libroc_gen_dev.rlib
libroc_gen_llvm.d
libroc_gen_llvm.rlib
libroc_gen_wasm.d
libroc_gen_wasm.rlib
libroc_glue.d
libroc_glue.rlib
libroc_highlight.d
libroc_highlight.rlib
libroc_ident.d
libroc_ident.rlib
libroc_intern.d
libroc_intern.rlib
libroc_late_solve.d
libroc_late_solve.rlib
libroc_linker.d
libroc_linker.rlib
libroc_load.d
libroc_load.rlib
libroc_load_internal.d
libroc_load_internal.rlib
libroc_module.d
libroc_module.rlib
libroc_mono.d
libroc_mono.rlib
libroc_packaging.d
libroc_packaging.rlib
libroc_parse.d
libroc_parse.rlib
libroc_problem.d
libroc_problem.rlib
libroc_region.d
libroc_region.rlib
libroc_repl_cli.d
libroc_repl_cli.rlib
libroc_repl_eval.d
libroc_repl_eval.rlib
libroc_repl_expect.d
libroc_repl_expect.rlib
libroc_repl_wasm.d
libroc_repl_wasm.dylib
libroc_reporting.d
libroc_reporting.rlib
libroc_serialize.d
libroc_serialize.rlib
libroc_solve.d
libroc_solve.rlib
libroc_solve_problem.d
libroc_solve_problem.rlib
libroc_target.d
libroc_target.rlib
libroc_test_utils.d
libroc_test_utils.rlib
libroc_tracing.d
libroc_tracing.rlib
libroc_types.d
libroc_types.rlib
libroc_unify.d
libroc_unify.rlib
libroc_utils.d
libroc_utils.rlib
libroc_wasm_interp.d
libroc_wasm_interp.rlib
libroc_wasm_module.d
libroc_wasm_module.rlib
libtest_gen.d
libtest_gen.rlib
libven_graph.d
libven_graph.rlib
libven_pretty.d
libven_pretty.rlib
libwasi_libc_sys.d
libwasi_libc_sys.rlib
```